packages/fairroot/package.py

```diff
@@ -40,6 +40,8 @@ class Fairroot(CMakePackage):
     # Dependencies for dev version
     depends_on('boost@1.68.0: cxxstd=11 +container')
 
+    depends_on('vmc', when='@dev')
+
```

Comment:
Has this changed in fairroot?
Or did the old versions need flatbuffers also, just we didn't specifiy it here?

Reply:
Okay, my fault. No idea why I haven't seen the `@dev`.